contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -2912,3 +2912,8 @@ END;
 $BODY$
 LANGUAGE plpgsql;
 GRANT EXECUTE ON FUNCTION sys.INDEXPROPERTY(IN object_id INT, IN index_or_statistics_name sys.nvarchar(128),  IN property sys.varchar(128)) TO PUBLIC;
+
+
+CREATE OR REPLACE FUNCTION sys.schema_name(IN id oid) RETURNS VARCHAR
+AS 'babelfishpg_tsql', 'schema_name' LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
+GRANT EXECUTE ON FUNCTION sys.schema_name(IN oid) TO PUBLIC;
```

Comment:
We don't need to move the schema_name function anymore because we use an alternative method of obtaining schema name